notebooks/examples/join_duckdb.py

```diff
@@ -0,0 +1,32 @@
+import sys
```

Comment:
Love having the example files here! Consider adding a docstring describing the purpose to make this explicit

Reply:
Great point, thanks!